problems/scope/problem_nb-no.md

```diff
@@ -0,0 +1,74 @@
+---
+
+# SCOPE
+
+`Scope` er de variablene, objektene og funksjonene du har tilgang til.
+
+JavaScript har to scope: `global` og `lokal`. En variabel som er deklarert utenfor en funksjon er en `global` variabel. Dens verdi er tilgjengelig og kan endres gjennom hele programmet ditt. En variabel som er deklarert inni en funksjon er `lokal`. Den lages og fjernes hver gang funksjonen kjøres og variabelen kan ikke nås av kode som er utenfor funksjonen.
+
+Funksjoner som er definert inni andre funksjoner, kjent som nøstede funksjoner, har tilgang til scopet til den ytre funksjonen den er deklarert i.
+
+Følg nøye med på kommentarene i koden under:
+
+```js
+var a = 4;  // a er en global variabel, den kan nås av funksjonene under
+
+function foo() {
+    var b = a * 3;  // b kan ikke nås utenfor foo funksjonen, men kan nås av funksjoner
+                    // definert inni foo
+
+    function bar(c) {
+        var b = 2;  // enda en `b` variabel blir lagd i bar funksjonens scope
+                    // endringer på den nye `b` variabelen endrer ikke den ytre `b` variabelen
+        console.log( a, b, c );
+    }
+
+    bar(b * 4);
+}
+
+foo(); // 4, 2, 48
+```
+IIFE, Immediately Invoked Function Expression, er et pattern for å lage lokale scope
```

Comment:
En god norsk oversettelse for `IIFE` hoppet jeg over fordi (a) fant ikke et alternativ som hørtes greit ut og (b) det er et uttrykk som mange bruker og derfor burde læres selv om det er engelsk. Enig?